embed-code-go/embedding/parsing/parsing_context.go

```diff
@@ -147,6 +148,14 @@ func (pc *ParsingContext) ResolveEmbeddingNotFound() {
 	pc.EmbeddingsNotFound = append(pc.EmbeddingsNotFound, currentEmbedding.Embedding)
 }
 
+// Writes the source content of the markdown file if embedding is not accepted.
+func (pc *ParsingContext) ResolveEmbeddingNotAccepted() {
```

Comment:
Actually this method doesn't have any logic in it's name and doc. Maybe this should be two methods where one writes markdown while the other performs some updates over `Embedings`. 
Also from the code perspective looks like this method doesn't actually `writes` markdown instead it just remove potential embedding from the list. Please review method and doc.